src/input/mpegts/linuxdvb/linuxdvb_ddci.c

```diff
@@ -144,6 +153,18 @@ linuxdvb_ddci_mtimer_arm_rel
     pthread_mutex_unlock(&global_lock);
 }
 
+/* returns 1, if the given PID shall be sent to the CAM AND to the service */
+static inline int
+linuxdvb_ddci_cam_pid_required ( int_fast16_t pid )
```

Comment:
I don't like this type of filtering.

Reply:
See my comment for the other code snippet below.